lib/Receiver.js

```diff
@@ -36,10 +36,8 @@ class Receiver {
     this.expectHandler = null;
     this.currentMessage = [];
     this.currentMessageLength = 0;
-    this.messageHandlers = [];
     this.expectData(2, this.processPacket);
     this.dead = false;
-    this.processing = false;
 
     this.onerror = function () {};
```

Comment:
Unrelated to this change but I think it makes sense to initialize these handlers to `null` or make them point to the same `noop` function. It doesn't make sense to use 6 different `noop` functions.

I think it's fine to do this in this PR without opening a new one.